### ISSUE TITLE
[Mod Compatibility] CreatureLevelAndLootControl

### ISSUE THREAD

**Issue by OrbitalHustler** [Grantapher/ValheimPlus#101]
- **OS:** Windows Client
- **Valheim Game Version:** 0.218.21
- **V+ Mod Version:** 0.9.14.0

**Describe the bug**

CreatureLevelAndLootControl fails to load because of Valheim Plus:
[Error  :   BepInEx] Could not load [Creature Level & Loot Control 4.6.1] because it is incompatible with: org.bepinex.plugins.valheim_plus

**To Reproduce**

Install CreatureLevelAndLootControl v4.6.1: https://thunderstore.io/c/valheim/p/Smoothbrain/CreatureLevelAndLootControl/

**Expected behavior**

Both mods load properly

**Troubleshooting**

CreatureLevelAndLootControl cannot load

**Comment by Grantapher** [comment]
Yeah it is on purpose from Blaxxun's side. Still don't know what their problem is, something with the old mod author(s?) of V+.

https://github.com/blaxxun-boop/PassivePowers/issues/5
